http/swagger.yml

```diff
@@ -5395,6 +5395,9 @@ paths:
       summary: List all users
       parameters:
         - $ref: "#/components/parameters/TraceSpan"
+        - $ref: "#/components/parameters/Offset"
+        - $ref: "#/components/parameters/Limit"
+        - $ref: "#/components/parameters/After"
```

Comment:
These swagger updates should be made to the openapi repo

Reply:
Changes are moved into `openapi` repository: https://github.com/influxdata/openapi/pull/44